### PR TITLE
microsoft-teams 23247.720.2421.8365

### PR DESCRIPTION
**Pull request by sergiotocalini**
This PR follows the announcement of the new Microsoft Teams version.

https://www.microsoft.com/en-us/microsoft-teams/download-app

It has an open discussion as well: https://github.com/orgs/Homebrew/discussions/4834

### CODE REVIEW

File: Casks/m/microsoft-teams-classic.rb [hunk 1]
```diff
@@ -0,0 +1,54 @@
+cask "microsoft-teams-classic" do
+  version "1.6.00.27656"
+  sha256 "cd4cc25e0b4d24037e414226e201176e5be5d350d4d35be73fc531f11bc7ad2a"
+
```
Comment (SMillerDev) [inline Casks/m/microsoft-teams-classic.rb:4]: This should be in `cask-versions` or at the very least in a new PR.

**Comment by miccal** [comment]
FYI: I have been using the new version of Teams for a while:

https://github.com/miccal/homebrew-miccal/blob/master/Casks/m-microsoft-teams.rb

I have this installed with the standard `microsoft-teams` Cask, and they do not conflict with each other as far as I can see.

(And in typical Microsoft fashion, the new one is even more terrible than the old one.)

**Comment by p-linnane** [comment]
The Classic one should live in `homebrew-cask-versions`. Good with this otherwise.

**Comment by miccal** [comment]
https://github.com/Homebrew/homebrew-cask-versions/pull/17982

**Review by p-linnane — APPROVED** [review]

**Comment by muescha** [comment]
it not show up in `brew outdated`
microsoft teams 1.6.x do not ask for an upgrade itself

whats the best "upgrade" path? do a force reinstall?

**Comment by zauni** [comment]
@muescha In my case it looks like Homebrew did not recognize that Teams was renamed to "Teams classic" or something like that, and it thought I did not install it at all (`brew ls` did not show it). So a normal `brew install microsoft-teams` was enough to get the new Teams version installed alongside the Classic Teams.

**Comment by mxmo0rhuhn** [comment]
Hi guys, it's great to see that the new teams made it to homebrew so fast! 
I have a question regarding the `microsoft-office-businesspro` bundle (https://github.com/Homebrew/homebrew-cask/blob/90b10375bd45de9798f3f392b388f6671cbd9912/Casks/m/microsoft-office-businesspro.rb): Do I understand it correctly that this cask should now also conflict with the classic teams? 
Moreover: Do you have any indication when/how the new teams will be available in this bundle?

**Comment by miccal** [comment]
https://github.com/Homebrew/homebrew-cask/pull/157915

**Pull request merged**